robottelo/ui/locators/base.py

```diff
@@ -943,6 +944,10 @@
                           "//input[@placeholder='Filter permissions']"),
     "roles.perm_type": (By.XPATH, "//label[contains(., '%s')]"),
     "roles.permission": (By.XPATH, "//input[@value='%s']"),
+    "roles.resources": (
+        By.XPATH, "//table[contains(@id, 'DataTables_Table')]/tbody/tr/td[1]"),
```

Comment:
The idea of this locator was to fetch all cells from the first column that represents resource types. Unfortunately, rows and columns haven't any specific class, id or any other attribute. The only alternative is to hardcode header name and calculate its position, like: `//table[contains(@id, 'DataTables')]/tbody/tr/td[count(//table/thead/tr/th[.="Resource"]/preceding-sibling::th)+1]`